pkg/network/controllers/vm.go

```diff
@@ -154,3 +123,27 @@ func (v *VMController) vmiInterfacesPatch(newVmiSpec *v1.VirtualMachineInstanceS
 
 	return err
 }
+
+func hasOrdinalSecondaryIfaces(
+	networks []v1.Network,
+	ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface,
+) bool {
+	secondaryNets := vmispec.FilterMultusNonDefaultNetworks(networks)
+	if len(secondaryNets) == 0 {
+		return false
+	}
+
+	ifaceStatusesByName := vmispec.IndexInterfaceStatusByName(ifaceStatuses, nil)
+	for _, net := range secondaryNets {
+		ifaceStatus, ok := ifaceStatusesByName[net.Name]
+		if !ok || ifaceStatus.PodInterfaceName == "" {
+			continue
```

Comment:
nit: I would prefer not to use this "continue", it is hard to read. If you can condition for the early return, it will be nicer.

You actually could have used the predicated to filter the index (instead of passing the `nil`).

Reply:
Dropped the continue statement.